src/utils/hex.cairo

```diff
@@ -34,6 +34,27 @@ fn hex_char_to_nibble(hex_char: u8) -> u8 {
     }
 }
 
+/// Print a byte array as hex
+pub fn print_bytearray_hex(data: @ByteArray) {
```

Comment:
If you want to keep is a part of this PR i'd suggest to refactor this as `to_hex(data: @ByteArray) -> ByteArray` and add a test.

The you can use it as `println!("{}", to_hex(...))` which is more flexible and intuitive